msal/src/test/res/raw/single_account_dcf_test_config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "client_id": "d335d85e-5aa7-4f03-8c2f-81b2699f30a6",
+  "authorization_user_agent": "DEFAULT",
+  "redirect_uri": "msauth://com.onmicrosoft.fadiorgorg/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
```

Comment:
Can we use something that is already existing in the LAB?

Reply:
You mean one of the other .json files that was already there?

Reply:
Yes, can we use one of those? Does support for DCF need to be turned on per tenant by the tenant admin? If yes, then we should start a thread with LAB Team to configure something for this. cc/ @rpdome

Reply:
Yes, DCF has to be turned on. If no one has started the thread, let's create a work item and we can pick that up later.